lib/web_ui/lib/src/engine/canvaskit/image_filter.dart

```diff
@@ -26,28 +34,62 @@ class CkImageFilter extends ManagedSkiaObject<SkImageFilter> implements ui.Image
   void delete() {
     rawSkiaObject?.delete();
   }
+}
+
+class _CkColorFilterImageFilter extends CkImageFilter {
+  _CkColorFilterImageFilter({ required this.colorFilter }) : super._();
+
+  final CkColorFilter colorFilter;
+
+  @override
+  SkImageFilter _initSkiaObject() => colorFilter._rawImageFilter;
+
+  @override
+  int get hashCode => colorFilter.hashCode;
 
+  @override
+  bool operator ==(Object other) {
+    if (runtimeType != other.runtimeType)
```

Comment:
I think `_CkColorFilterImageFilter != other.runtimeType` would be faster (but only correct if this class doesn't have subclasses)

(this comment applies here and in other implementations of `operator==`)

Reply:
see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#common-boilerplates-for-operator-and-hashcode